### ISSUE TITLE
Background shading behind arrival time cards isn't consistent

### ISSUE THREAD

**Issue by barbeau** [OneBusAway/onebusaway-android#451]
**Summary:** 

If you look closely at the background color behind the cards for Arrival info, both Style A and B, there is a darker gray surrounding by a lighter gray - the background should all be the same color (minus the shadow of the card itself).

This seems to be more noticeable when viewing the ArrivalsListFragment within the ArrivalsListActivity than the sliding panel, but exists in both views.

I believe this started showing up when my device (LG G4) updated to Android 6.0.

**Steps to reproduce:** 

For ArrivalsListActivity - star a stop, and then tap on the starred stop from the Starred Stops view.  Or, create a home screen shortcut and tap on that shortcut.

For sliding panel - go to Nearby view, and tap on a bus stop and slide the panel up.

**Expected behavior:** 

The color behind the cards should be one shade of gray (minus the shadow from the card itself).

**Observed behavior:** 

There are two shades of gray - a darker one surrounding the card, and lighter one in a square surrounding the darker shade.

**Device and Android version:** 

Stock LG G4 w/ Android 6.0.  As mentioned above, I don't believe this occurred on 5.x on the same device.

In OBA project dependencies:

`compile 'com.android.support:cardview-v7:22.2.1'`
 `compile 'com.android.support:design:22.2.1'`

**Screenshots:** 

![image](https://cloud.githubusercontent.com/assets/928045/14086297/6cb13d30-f4f2-11e5-80a5-7be09a1f5daf.png)

**Comment by barbeau** [comment]
This is also very noticeable in the ArrivalsListActivity view for stops with no immediate upcoming arrivals:

![image](https://cloud.githubusercontent.com/assets/928045/14086782/e56b9034-f4f4-11e5-89e7-4f80572d1091.png)

It looks like the fill color within the card itself is off as well.

**Comment by barbeau** [comment]
From a quick look, it seems that this is still a problem with:

`compile 'com.android.support:cardview-v7:23.4.0'`
`compile 'com.android.support:design:23.4.0'`

**Comment by kungharrison** [comment]
I think I found the issue, I was able to remove the dark gray behind the cards if that's what is intended. The problem seems to be that there are unique color resources for each list's background with slightly different off-white colors compared to the plain white background (i.e. arrival_list_background is a slight off-white color), and that the background colors are only being applied to the footer's style instead of the entire background. That can be fixed easily, but is it intentional that all the background color resources used are slightly different colors of white?  

![image](https://github.com/OneBusAway/onebusaway-android/assets/53415491/51196e4b-3149-40ae-975b-a1d9b630e17b)

ArrivalsListActivity view:
![image](https://github.com/OneBusAway/onebusaway-android/assets/53415491/765a4b19-f922-45bb-bc07-8c4086d5ae4b)
